_data/security.yml

```diff
@@ -127,6 +127,13 @@ websites:
       twitter: gogetssl
       facebook: gogetssl
 
+    - name: HackerOne
+      url: https://hackerone.com
+      img: hackerone.png
+      tfa: Yes
+      software: Yes
+      doc: https://docs.hackerone.com/hackers/two-factor-authentication.html
+
```

Comment:
Maybe also add `twitter` and `facebook`, too?

Reply:
@reedloden not if `tfa` is set to Yes. See contributing.md